clojure-mode.el

```diff
@@ -2063,9 +2056,8 @@ many times."
         (condition-case nil
             (save-match-data
               (let ((original-position (point))
-                    clojure-comment-start clojure-comment-end)
+                    clojure-comment-end)
                 (beginning-of-defun)
-                (setq clojure-comment-start (point))
```

Comment:
Why did you change this?

Reply:
`clojure-comment-start` was generating a warning and as far as I can tell It is unused? It may be that I'm missunderstanding something?
```
clojure-mode.el:2023:1:Warning: Unused lexical variable
    ‘clojure-comment-start’
```

Reply:
Hmm, I think that's supposed to be a buffer local variable. Let me take a closer look at the code - we might have made some real mistake. :-)